gradio/helpers.py

```diff
@@ -0,0 +1,780 @@
+"""
```

Comment:
Nice to have all of the component helpers in one place.

Do you think we should add `gr.Request` and `mount_gradio_app()` here as well? I don't think a separate "Routes" section makes sense

Reply:
PR is already huge, can do cleanup in another pr